api/envoy/extensions/filters/http/grpc_field_extraction/v3/config.proto

```diff
@@ -83,7 +97,7 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // }
 
 message GrpcFieldExtractionConfig {
-  // DescriptorSet is in local data source. It could be either in a local file or embedded in the
+  // descriptor_set is in local data source. It could be either in a local file or embedded in the
```

Comment:
This still doesn't say exactly what is going to be returned from this DataSource.  For example, let's say that the DataSource just points to a local file.  If I were someone looking at this doc and trying to figure out how to configure this filter, how would I know what contents to put in the file?

Reply:
Updated.